docs/release-notes/2.15.0.md

```diff
@@ -0,0 +1,95 @@
+
+
+
+## Improvements
+
+* [PMM-6693](https://jira.percona.com/browse/PMM-6693): pmm-managed denies request when using API keys even if the role set is admin -- The PMM API will now accept requests not only with username/password but also with API tokens generated in Grafana. 
+* [PMM-7487](https://jira.percona.com/browse/PMM-7487): Check connection to HAProxy services when adding them for monitoring (UI) (Thanks to Jiří Čtvrtka for reporting this issue)
+* [PMM-7476](https://jira.percona.com/browse/PMM-7476): Improve `pmm-admin --help` output, highlight required fields for external services
+* [PMM-7473](https://jira.percona.com/browse/PMM-7473): Improve descriptions of `pmm-admin` flags and required fields for external services
+* [PMM-7471](https://jira.percona.com/browse/PMM-7471): Update VictoriaMetrics 1.52.0 to 1.53.1 (See [VictoriaMetrics 1.53.0](https://github.com/VictoriaMetrics/VictoriaMetrics/releases/tag/v1.53.0) and [VictoriaMetrics 1.53.1](https://github.com/VictoriaMetrics/VictoriaMetrics/releases/tag/v1.53.1).)
+* [PMM-7433](https://jira.percona.com/browse/PMM-7433): Integrated Alerting: Easier-to-read rule details in Alert Rules list
+* [PMM-7422](https://jira.percona.com/browse/PMM-7422): Security Threat Tool: View all active and silenced alerts
+* [PMM-7331](https://jira.percona.com/browse/PMM-7331): Security Threat Tool: Failed checks have 'Read more' links with helpful content
+* [PMM-7329](https://jira.percona.com/browse/PMM-7329): Security Threat Tool: Hide check results tab if STT is disabled
+* [PMM-7328](https://jira.percona.com/browse/PMM-7328): Security Threat Tool: Download and run checks when activated, immediately, repeating every 24 hours thereafter (Previously, downloading and running new checks happened every 24 hours but the cycle didn't begin when STT was activated.)
+* [PMM-7259](https://jira.percona.com/browse/PMM-7259): Integrated Alerting: 'Alerting is disabled' message gives hint to enable it
+* [PMM-7257](https://jira.percona.com/browse/PMM-7257): Integrated Alerting: Easier-to-read rule details in Alert Rules list (API to expose Alert Rule's expression and real values)
+* [PMM-7240](https://jira.percona.com/browse/PMM-7240): DBaaS: Change from Dashboard to Grafana Page -- We changed the DBaaS page from a Grafana Dashboard to a Grafana Page to be better aligned with the DBaaS enable/disable status and avoid confusion when DBaaS is disabled.
+* [PMM-6711](https://jira.percona.com/browse/PMM-6711): Add filter for `external_group` in `pmm-managed` services list (API)
+* [PMM-6695](https://jira.percona.com/browse/PMM-6695): Update Grafana 7.1.3 to 7.3.7 (See [What's new in Grafana 7.2](https://grafana.com/docs/grafana/latest/whatsnew/whats-new-in-v7-2) and [What's new in Grafana 7.3](https://grafana.com/docs/grafana/latest/whatsnew/whats-new-in-v7-3/).)
+* [PMM-5888](https://jira.percona.com/browse/PMM-5888): Improve `pmm-admin --help` descriptions for external services
```

Comment:
How is this materially different from PMM-7476?

Reply:
removed  https://jira.percona.com/browse/PMM-7476 
Everything  will be in one  issue